packages/workflows/src/definition/product/update-products.ts

```diff
@@ -32,6 +33,11 @@ export const updateProductsWorkflowSteps: TransactionStepsDefinition = {
     next: {
       action: UpdateProductsActions.updateProducts,
       next: [
+        {
+          action: UpdateProductVariantsActions.upsertPrices,
+          noCompensation: true,
```

Comment:
question: We've configured the step to not run the compensating flow, yet we have a compensation handler in the `handlers`. Would think one of the two should be removed?